deploy/helm/templates/configmaps/trivy-operator-config.yaml

```diff
@@ -20,6 +20,7 @@ data:
   OPERATOR_METRICS_FINDINGS_ENABLED: {{ .Values.operator.metricsFindingsEnabled | quote }}
   OPERATOR_METRICS_VULN_ID_ENABLED: {{ .Values.operator.metricsVulnIdEnabled | quote }}
   OPERATOR_HEALTH_PROBE_BIND_ADDRESS: ":9090"
+  OPERATOR_PPROF_BIND_ADDRESS: {{ "" | quote }}
```

Comment:
Would it make sense to add support for configuring this flag via a Helm value?
it allows to use prepared configs https://github.com/afdesk/to-values/blob/main/pprof.yaml

Reply:
ah that's right! - I added 57fd7db715adbe4949272b2446763ad9de62fbec